.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - 0.10
   - 4
   - 6
   - 8
```

Comment:
I'll be soon moving this repo from Travis to CircleCI but can we add node 10 and 12 as testing environments from now?

Reply:
Sounds great! added